apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConversionOptions.cs

```diff
@@ -55,10 +56,12 @@ internal class SpannerConversionOptions
         internal const string DecimalToPgNumeric = nameof(DecimalToPgNumeric);
         internal const string DateTimeToDate = nameof(DateTimeToDate);
         internal const string DateTimeToTimestamp = nameof(DateTimeToTimestamp);
+        internal const string GuidToUuid = nameof(GuidToUuid);
```

Comment:
I don't think we need these options for Guid. The relantionship between the Spanner and CLR type is one to one here.